protostar/composition_root.py

```diff
@@ -158,10 +161,14 @@ def build_di_container(script_root: Path):
             log_color_provider=log_color_provider,
         ),
     ]
+    toml_version_checker = ProtostarTOMLVersionChecker(
+        protostar_toml_reader=protostar_toml_reader, version_manager=version_manager
+    )
 
     protostar_cli = ProtostarCLI(
         commands=commands,
         latest_version_checker=latest_version_checker,
+        toml_version_checker=toml_version_checker,
```

Comment:
This name is confusing. It suggests checking the version of any `toml`. I'd change all occurrences of `toml_version_checker`.

```suggestion
        protostar_toml_version_checker=protostar_toml_version_checker,
```